java/app/src/main/java/com/fullstorydev/shoppedemo/adapters/CheckoutBindingAdapter.java

```diff
@@ -1,12 +1,14 @@
 package com.fullstorydev.shoppedemo.adapters;
 
-import android.widget.Spinner;
```

Comment:
MDC does not have built in `Spinner`, they advise to use `AutoCompleteTextView` in stead